package.json

```diff
@@ -22,8 +22,11 @@
     "components:build-cdn:versions-list": "node tasks/update-versions-list.js",
     "components:publish-cdn": "node tasks/publish-cdn.js",
     "components:check-i18n": "node tasks/check-i18n.js",
+    "components:check-type-imports": "node tasks/check-type-imports.js",
     "components:check-lit": "lit-analyzer src",
     "components:docs": "cem analyze --litelement --outdir dist",
+    "test-cem": "mocha cem/test/*.js",
+    "test-cem:watch": "mocha --watch cem/test/*.js",
```

Comment:
Should we use `test:cem` and `test:cem:watch` to match the other tasks? WDYT? :thinking:

Reply:
Seems fair :wink: